### PR TITLE
Legacy REP faucet and migration page

### PR DESCRIPTION
**Pull request by nuevoalex**
[Clubhouse Story](https://app.clubhouse.io/augur/story/7978)

## Submitter checklist
- [X] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [X] Steps to reproduce
  - [X] Screenshot (If applicable)

## Reviewer Checklist
- [x] Test/lint pass 
- [ ] Post merge, story marked complete

### CODE REVIEW

File: src/modules/account/components/account-legacy-rep/account-legacy-rep.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import { augur } from 'services/augurjs'
+
+import Styles from 'modules/account/components/account-legacy-rep/account-legacy-rep.styles'
+
+export default class AccountLegacyRep extends Component {
```
Comment (justinbarry) [inline src/modules/account/components/account-legacy-rep/account-legacy-rep.jsx]: Can this be a stateless component?

Comment (nuevoalex) [reply src/modules/account/components/account-legacy-rep/account-legacy-rep.jsx]: maybe? would the format for the functions provided in the props still function if they were just referenced directly? I imagine so but I'm depending on a lot of magic boilerplate everywhere so I'm not sure.

Comment (nuevoalex) [reply src/modules/account/components/account-legacy-rep/account-legacy-rep.jsx]: Yup, seems fine! Will change

**Comment by justinbarry** [comment]
```
client.js?3ac5:183 [HMR] bundle has 6 warnings
client.js?3ac5:184 ./src/modules/reporting/components/reporting-dispute-progress/reporting-dispute-progress.jsx
12:30-63 "export 'calculateTentativeStakePercentage' was not found in 'modules/reporting/helpers/progress-calculations'
 @ ./src/modules/reporting/components/reporting-dispute-progress/reporting-dispute-progress.jsx
 @ ./src/modules/reporting/components/reporting-dispute-form/reporting-dispute-form.jsx
 @ ./src/modules/reporting/components/reporting-dispute/reporting-dispute.jsx
 @ ./src/modules/reporting/containers/reporting-dispute.js
 @ ./src/modules/routes/constants/components.js
 @ ./src/modules/routes/components/routes/routes.jsx
 @ ./src/modules/app/components/app/app.jsx
 @ ./src/modules/app/containers/app.js
 @ ./src/main.jsx
 @ multi react-hot-loader/patch webpack-hot-middleware/client ./src/main
./src/modules/app/components/app/app.jsx
227:15-33 "export 'ACCOUNT_LEGACY_REP' was not found in 'modules/routes/constants/views'
 @ ./src/modules/app/components/app/app.jsx
 @ ./src/modules/app/containers/app.js
 @ ./src/main.jsx
 @ multi react-hot-loader/patch webpack-hot-middleware/client ./src/main
./src/modules/account/components/account-view/account-view.jsx
31:65-83 "export 'ACCOUNT_LEGACY_REP' was not found in 'modules/routes/constants/views'
 @ ./src/modules/account/components/account-view/account-view.jsx
 @ ./src/modules/routes/constants/components.js
 @ ./src/modules/routes/components/routes/routes.jsx
 @ ./src/modules/app/components/app/app.jsx
 @ ./src/modules/app/containers/app.js
 @ ./src/main.jsx
 @ multi react-hot-loader/patch webpack-hot-middleware/client ./src/main
./src/modules/app/components/inner-nav/account-inner-nav.jsx
41:51-69 "export 'ACCOUNT_LEGACY_REP' was not found in 'modules/routes/constants/views'
 @ ./src/modules/app/components/inner-nav/account-inner-nav.jsx
 @ ./src/modules/app/components/app/app.jsx
 @ ./src/modules/app/containers/app.js
 @ ./src/main.jsx
 @ multi react-hot-loader/patch webpack-hot-middleware/client ./src/main
./src/modules/app/components/inner-nav/account-inner-nav.jsx
43:20-38 "export 'ACCOUNT_LEGACY_REP' was not found in 'modules/routes/constants/views'
 @ ./src/modules/app/components/inner-nav/account-inner-nav.jsx
 @ ./src/modules/app/components/app/app.jsx
 @ ./src/modules/app/containers/app.js
 @ ./src/main.jsx
 @ multi react-hot-loader/patch webpack-hot-middleware/client ./src/main
./src/modules/app/components/app/app.jsx
69:553-571 "export 'ACCOUNT_LEGACY_REP' was not found in 'modules/routes/constants/views'
 @ ./src/modules/app/components/app/app.jsx
 @ ./src/modules/app/containers/app.js
 @ ./src/main.jsx
 @ multi react-hot-loader/patch webpack-hot-middleware/client ./src/main
```

**Comment by justinbarry** [comment]
NM. Freaking webpack.

**Pull request merged**